modules/installation-configuration-parameters.adoc

```diff
@@ -405,6 +409,80 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 
 endif::gcp[]
 
+ifdef::vsphere[]
+.Additional VMware vSphere cluster parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.vsphere.vCenter`
+|The fully-qualified host name or IP address of the vCenter server.
+|String
+
+|`platform.vsphere.username`
+|The user name to use to connect to the vCenter instance with. This user must have at least
+the roles and privileges that are required for
+link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/vcp-roles.html[static or dynamic persistent volume provisioning]
+in vSphere.
+|String.
+
+|`platform.vsphere.password`
+|The password for the vCenter user name.
+|String.
+
+|`platform.vsphere.datacenter`
+|The name of the datacenter to use in the vCenter instance.
+|String.
+
+|`platform.vsphere.defaultDatastore`
+|The name of the default datastore to use for provisioning volumes.
+|String.
+
+|`platform.vsphere.folder`
+|_Optional_. The absolute path of an existing folder where the installation program creates the virtual machines. create VMs. If you do not provide this value, the installation program creates a folder that is named with the cluster ID is created in the datacenter virtual machine folder.
+|String, for example, `/datacenter/vm/folder/subfolder`.
```

Comment:
if user customized folder, it's better to describe the format of the absolute folder path: /<datacenter name>/vm/<folder_name>/<sub_foldername>

Reply:
Sorry I missed datacenter_name in the format, should be `/<datacenter_name>/vm/<folder_name>/<sub_foldername>`